static/js/components/source/source_publish/SourcePublish.jsx

```diff
@@ -47,35 +48,35 @@ const SourcePublish = ({ sourceId, isElements }) => {
     text: "Publish",
     color: "",
   });
+  const [sourcePublishReleaseOpen, setSourcePublishReleaseOpen] =
```

Comment:
To encourage people to pick a release, I think we should have the release tab opened by default at least.